bounties_api/analytics/management/commands/timeline_generator.py

```diff
@@ -165,17 +165,17 @@ def get_noise_bounties(bounties):
     return noise
 
 
-def generate_timeline(time_frame, schema):
+def generate_timeline(time_frame, schema, platform):
     date = time_frame[1]
     bounty_state_schema = BountyState.objects
     fulfillment_schema = Fulfillment.objects
 
-    if schema == 'standardSchema':
-        bounty_state_schema = bounty_state_schema.filter(DEFAULT_SCHEMA_QUERY)
-        fulfillment_schema = fulfillment_schema.filter(DEFAULT_SCHEMA_QUERY)
-    elif schema and schema != ALL_SCHEMA:
-        bounty_state_schema = bounty_state_schema.filter(bounty__schemaName=schema)
-        fulfillment_schema = fulfillment_schema.filter(bounty__schemaName=schema)
+    if platform and platform != ALL_PLATFORM:
```

Comment:
I replaced it with platform.  The new logic filters by platform if a platform is provided and it isn't ALL_PLATFORM, otherwise it defaults by the default platform, 'bounties-network'

I don't think ALL_SCHEMA will still be needed if we completely replace schema with platform. Or would it?